components/raftstore-v2/src/operation/life.rs

```diff
@@ -260,7 +306,223 @@ impl Store {
     }
 }
 
+/// Tell leader that `to_peer` is destroyed.
+fn report_peer_destroyed(tombstone_msg: &mut RaftMessage) -> Option<RaftMessage> {
```

Comment:
minor: I think the fn name could be renamed as create_raft_tombstone_message(), as there's no report logic in this function at all.